### PR TITLE
fix(Dialog): correct width's vars order

### PR DESCRIPTION
**Pull request by saxumcordis**
it is necessary to reverse the order of variables to give opportunity to change dialog width

### CODE REVIEW

**Review by amje — APPROVED** [review]

File: src/components/Dialog/Dialog.scss [hunk 1]
```diff
@@ -9,7 +9,7 @@ $block: '.#{variables.$ns}dialog';
     position: relative;
     display: flex;
     flex-direction: column;
-    width: var(--_--width, var(--g-dialog-width));
```
Comment (saxumcordis) [inline src/components/Dialog/Dialog.scss:12]: why here is double var()

its ok to use `var(--g-dialog-width), --_--width)`, isn't it?

@amje

File: src/components/Dialog/Dialog.scss [hunk 2]
```diff
@@ -9,7 +9,7 @@ $block: '.#{variables.$ns}dialog';
...
+    width: var(var(--g-dialog-width), --_--width);
```
Comment (ValeraS) [inline src/components/Dialog/Dialog.scss:12]:
```suggestion
    width: var(--g-dialog-width, var(--_--width));
```

**Review by ValeraS — CHANGES_REQUESTED** [review]

File: src/components/Dialog/Dialog.scss (hunk 1 above)
Comment (ValeraS) [reply src/components/Dialog/Dialog.scss:12]: https://developer.mozilla.org/en-US/docs/Web/CSS/var#syntax

**Comment by saxumcordis** [comment]
not actual

File: src/components/Dialog/Dialog.scss (hunk 1 above)
Comment (amje) [reply src/components/Dialog/Dialog.scss:12]: @saxumcordis No, @ValeraS is right, syntax is incorrect

**Review by ValeraS — APPROVED** [review]

**Pull request merged**